src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/CapabilitiesTest.java

```diff
@@ -795,6 +796,223 @@ public void testNonAdvertisedNonQueriableWithinGroup() throws Exception {
         }
     }
 
+    @Test
+    public void testRootLayerRemovedWMSService() throws Exception {
+        // make layers non advertised
+        WMSInfo info = getWMS().getServiceInfo();
+        info.getMetadata().put(WMS.NO_ROOT_LAYER_IN_CAPABILITIES_KEY, true);
+        getGeoServer().save(info);
+        Catalog catalog = getCatalog();
+        setAdvertised(catalog, MockData.LAKES, false);
+        setQueryable(catalog, MockData.LAKES, false);
```

Comment:
I don't understand why the queriable flag has been switched off here? Does it affect layer visibility in the capabilities document?
Also, is making the layers not advertised changing the outcome? They are part of a "named" group, not a "single" group